blacklight.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
 
   s.add_dependency "rails",     ">= 3.2.6", "< 5"
   s.add_dependency "nokogiri",  "~>1.6"     # XML Parser
-  s.add_dependency "kaminari", "~> 0.15"  # the pagination (page 1,2,3, etc..) of our search results
+  s.add_dependency "kaminari", ">= 0.15", "< 1.1.0"   # the pagination (page 1,2,3, etc..) of our search results
```

Comment:
They already have bumped the version in master to 1.0.0.alpha.  I have this branch that I want to use: https://github.com/jcoyne/kaminari/tree/route_prefix_prototype